test/data/metadata/keywords2/metadata.yaml

```diff
@@ -0,0 +1,8 @@
+contact: The ACCESS Oracle
+email: oracle@example.com
+created: 2020-01-01
+description: Description again!
+notes: Notes
+keywords:
+  - cosima
+  - not-a-keyword
```

Comment:
Is this not an allowable keyword due to dashes?

Reply:
That's poor naming on my behalf!